src/Fcf.hs

```diff
@@ -96,6 +97,14 @@ module Fcf
   , Zip
   , Unzip
   , Cons2
+  , Concat
+  , ConcatMap
+  , Replicate
+  , Take
+  , Drop
+  , TakeWhile
+  , DropWhile
+  , Reverse
```

Comment:
I don't think `Fcf` should reexport `Fcf.Data.List`. What's there is just what was there at the start (when there was only one module) but otherwise I consider the module mostly frozen.

Reply:
ok, changed back to the original